wandb/sdk/launch/runner/local.py

```diff
@@ -136,14 +136,16 @@ def run(self, launch_project: LaunchProject) -> Optional[AbstractRun]:
         # updates to the tracking server when finished. Note that the run state may not be
         # persisted to the tracking server if interrupted
         if synchronous:
+            command_args += ["/bin/bash -c"]
+            command_args += ['"pip', "install", "-e", "./client", ";"]
```

Comment:
Is this only for devs?

Reply:
This needs to be removed from this PR. But this (along with the mounting of the cli) was the hack I found to have the launched run use my current dev cli version. Perhaps we can add a dev mode to do this later.